This patch comes from in-house.  It has not yet been submitted upstream,
but submission is planned.

--- Python-2.7.10/Lib/test/test_ssl.py.~1~	2015-05-10 10:17:04 -0800
+++ Python-2.7.10/Lib/test/test_ssl.py	2015-05-18 14:53:04 -0800
@@ -2222,7 +2219,7 @@
                 sys.stdout.write("\n")
             if hasattr(ssl, 'PROTOCOL_SSLv2'):
                 try:
-                    try_protocol_combo(ssl.PROTOCOL_SSLv23, ssl.PROTOCOL_SSLv2, True)
+                    try_protocol_combo(ssl.PROTOCOL_SSLv23, ssl.PROTOCOL_SSLv2, False)
                 except socket.error as x:
                     # this fails on some older versions of OpenSSL (0.9.7l, for instance)
                     if support.verbose:
